### PR TITLE
Remove pobox in profile page temporarily

### PR DESCRIPTION
**Pull request by jhflorey**

### CODE REVIEW

**Review by amber-emmes — APPROVED** [review]
Looks good.

**Review by Gbarra9 — APPROVED** [review]

**Review by amber-emmes — APPROVED** [review]
Looks good.

File: js/pages/settings.js [hunk 1]
```diff
@@ -432,18 +432,19 @@ const handleEditMailingAddressSection = () => {
...
     const zip = document.getElementById('UPAddress1Zip').value.trim();
-    const isPOBox = document.getElementById('poBoxCheckbox').checked;
+    // const isPOBox = document.getElementById('poBoxCheckbox').checked;
 
     const isMailingAddressValid = validateMailingAddress(1, addressLine1, city, state, zip);
     if (isMailingAddressValid) {
       formVisBools.isMailingAddressFormDisplayed = toggleElementVisibility(mailingAddressElementArray, formVisBools.isMailingAddressFormDisplayed);
       toggleButtonText();
-      submitNewMailingAddress(1, addressLine1, addressLine2, city, state, zip, isPOBox);
+    //   submitNewMailingAddress(1, addressLine1, addressLine2, city, state, zip, isPOBox);
```
Comment (JoeArmani) [inline js/pages/settings.js:441]: Please rm if this commented out is a duplicate line

**Review by JoeArmani — APPROVED** [review]
looks good, one minor comment

**Pull request merged**